src/main/java/frc/robot/commands/OuttakeCommand.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.IntakeSystem;
+
+public class OuttakeCommand extends Command {
+    private IntakeSystem intakeSystem;
```

Comment:
```suggestion
    private final IntakeSystem intakeSystem;
```

As with other comments about adding `final`, you should use `final` when possible for class members.

Reply:
done